website/content/commands/peering/generate-token.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: commands
+page_title: 'Commands: Peering Generate Token'
+---
+
+# Consul Peering Generate Token
+
+Command: `consul peering generate-token`
+
+Corresponding HTTP API Endpoint: [\[POST\] /v1/peering/token](/api-docs/peering#generate-a-peering-token)
+
+The `peering generate-token` generates a peering token.
+This token should be transferred to the other cluster being peered and consumed using [`consul peering establish`](/commands/peering/establish).
+
+Re-generating a token for a given name will not interrupt any active connection, but will
+invalidate any unused token for that name.
+
+The table below shows this command's [required ACLs](/api#authentication).
+
+| ACL Required |
+| ------------ |
+| `peering:write`  |
+
+## Usage
+
+Usage: `consul peering generate-token [options] -name <peer name>`
+
+#### Command Options
+
+- `-name=<string>` - (Required) The name of the peer.
+This name will only be used locally to refer to the connection.
+
+- `-meta=<string>=<string>` - Metadata to associate with the peering,
+formatted as key=value.
+This flag may be specified multiple times to set multiple metadata fields.
+
+- `-server-external-addresses=<string>[,string,...]` - A list of addresses to put into
+the generated token, formatted as a comma-separate list.
+This could be used to specify load balancer(s) or external IPs to reach the servers from
+the dialing side, and will override any server addresses obtained from the "consul" service.
```

Comment:
```suggestion
- `-server-external-addresses=<string>[,string,...]` - Specifies a comma-separated list of addresses to put into the generated token. You can specify one or more load balancers or external IPs that connect to Consul servers from the cluster that initiates the connection. As a result, Consul overrides any server addresses obtained from the `consul` service with this value.
```
Correct?

Reply:
To be consistent with my other suggestion.
```suggestion
- `-server-external-addresses=<string>[,string,...]` - Specifies a comma-separated list of addresses to put into the generated token. You can specify one or more load balancers or external IPs that route external traffic to this cluster's Consul servers. 
```